<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationDialog</class>
 <widget class="QDialog" name="ConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Game Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/icons.qrc">
    <normaloff>:/icons/preferences.png</normaloff>:/icons/preferences.png</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>431</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tabGame">
    <attribute name="title">
     <string>Game</string>
    </attribute>
    <widget class="QLabel" name="labelTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>11</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Title:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTitle">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelInitialState">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Initial State:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxInitialState">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>271</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>&lt;None&gt;</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tabVideo">
    <attribute name="title">
     <string>Video</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxVideoWindow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>291</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Window</string>
     </property>
     <widget class="QGroupBox" name="groupBoxVideoVideoWindowSize">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Size</string>
      </property>
      <widget class="QLabel" name="labelVindowWindowWidth">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>22</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Width:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="labelVindowWindowHeight">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>52</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Height:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxVideoWindowWidth">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>2560</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxVideoWindowHeight">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>2048</number>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="checkBoxVideoFullscreen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Fullscreen</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxVideoVsync">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Vsync</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxVideoCursor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cursor visible</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxVideo">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>291</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Video</string>
     </property>
     <widget class="QLabel" name="labelTargetFramerate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>22</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Target Framerate:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxVideoFramerate">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelAaLevel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Anti-Aliasing Level:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxVideoAaLevel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxTextureSheets">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>291</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Texture Sheets</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="labelSheetSize">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Sheet Size:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxTextureSheetSize">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1024</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2048</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4096</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8192</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16320</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButtonVideoTextureSheetsPreview">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Preview</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources/icons.qrc">
        <normaloff>:/icons/rainbow.png</normaloff>:/icons/rainbow.png</iconset>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabVersionInformation">
    <attribute name="title">
     <string>Version Information</string>
    </attribute>
    <widget class="QLabel" name="labelVersion">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Version:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditVersion">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelDescription">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>72</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelProduct">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>102</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Product:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelCompany">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>132</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Company:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelCopyright">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>162</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Copyright:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelTrademark">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>192</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Trademark:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDescription">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditProduct">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCompany">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCopyright">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTrademark">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxIncrementVersion">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>46</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Increment build</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>448</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEditJson">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>448</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit JSON</string>
   </property>
   <property name="icon">
    <iconset resource="../../resources/icons.qrc">
     <normaloff>:/icons/script.png</normaloff>:/icons/script.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
